src/components/slider/slider.js

```diff
@@ -0,0 +1,7 @@
+const slider = document.getElementById("slider");
+const sliderOutput = document.getElementById("slider-output");
+
+slider.addEventListener("change", (e) => {
+  console.log(slider);
+  sliderOutput.innerText = e.target.value;
+});
```

Comment:
looks good!!

Reply:
try to remove console.log from the js file , before deploying

Reply:
okay